Sources/Networking/Responses/RevenueCatUI/PaywallComponentsData.swift

```diff
@@ -73,6 +73,9 @@ public struct PaywallComponentsData: Codable, Equatable, Sendable {
         }
     }
 
+    // The unique identifier for this paywall
+    public let identifier: String
```

Comment:
@rickvdl Chatted with @aboedo during our 1:1 and we decided to call this... `deep_link_id` and we will `@_spi(Internal)` it 😊